insights/core/dr.py

```diff
@@ -414,7 +414,7 @@ def is_registry_point(component):
     return type(component).__name__ == "RegistryPoint"
 
 
-def get_registry_points(component):
+def get_registry_points(component, module='insights.specs.Specs'):
```

Comment:
@xiangce just a little nitpick, but looks like most of the strings here in the module defined with `"` double quotas and you seem to be defaulting to `'`.

Reply:
@vdymna - yup, we'd better keep the same style in the same module. I updated them. Thanks.